econplayground/main/models.py

```diff
@@ -474,3 +474,87 @@ class Meta:
 
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+
+class Question(models.Model):
+    adaptive = models.BooleanField(default=False)
+    ap_correct = models.ForeignKey(
+        'self',
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='ap_continue'
+    )
+    ap_incorrect = models.ForeignKey(
+        'self',
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='ap_intervene'
+    )
```

Comment:
I see this ap_correct, ap_incorrect stuff can work as the sort of tree structure we're looking for. Maybe this is all the structure we need to organize this! I'm not really sure what the best path forward is here, but I'm all for keeping it simple, and we can see how this works out.

Reply:
So don't implement the treebeard model?

Reply:
Then for the sake of time I'll leave it as is. It can always be changed later, and I need to jump on the actual tasks.